packages/broadcast/component.js

```diff
@@ -33,24 +32,57 @@ export class FabricBroadcast extends LitElement {
       return;
     }
     if (windowExists) {
-      await this.fetchMessage();
-      setInterval(() => this.fetchMessage(), this.interval);
+      await this._fetchMessage();
+      setInterval(() => this._fetchMessage(), this.interval);
     }
   }
 
-  async fetchMessage() {
+  async _fetchMessage() {
     const url = `${this.api}?path=${this.url}`;
     try {
       const res = await (await fetch(url)).json();
       this._messages = res.length ? res : [];
     } catch (err) {
-      console.error(`failed to fetch broadcasts from given url (${url})`)
+      console.error(`failed to fetch broadcasts from given url (${url})`);
     }
   }
 
+  get _broadcasts() {
+    if (!windowExists) return this._messages;
+    const ignoredBroadcasts = JSON.parse(window.localStorage.getItem('ignored-broadcasts')) || [];
+    return this._messages.filter(({ id }) => !ignoredBroadcasts.includes(id));
+  }
```

Comment:
The _broadcasts is a getter for use in the template that filters out any broadcasts that have been previously closed by the user.